examples/pytorch/mnist2/mnist.py

```diff
@@ -0,0 +1,205 @@
+# Copyright 2022 The Kubeflow Authors.
```

Comment:
Should this be PyTorch authors?

Reply:
I copied this over from the demos linked so I'm just taking what they listed. I can change of course.